src/redux/app/store.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import Bookreducer from '../feature/books/Bookslice';
+import CategoriesReducer from '../feature/categories/categoriesSlice';
+
+const store = configureStore({
+  reducer: {
+    Book: Bookreducer,
+    Categories: CategoriesReducer,
+  },
+});
+
+export default store;
```

Comment:
- Kindly make sure that the structure of files follows the duck pattern (see attached below):
   - [ ] Moving the `store.js` outside of the `app` folder and deleting the `app` folder.
   - [ ] Moving the `books` and `categories` folders outside the `feature` folder and deleting the `feature` folder.
![image](https://user-images.githubusercontent.com/62290545/221610947-cefcdd39-16ce-463a-80b9-902e21add21a.png)